Makefile

```diff
@@ -163,6 +163,19 @@ kfserving-install: knative-install cert-manager-install
 seldon-install: istio-install
 	@./scripts/seldon-operator-install.sh
 
+new-k8s-cluster:
+	@./scripts/ci/k3d.sh
+
+mlflow-e2e:
+	@./scripts/ci/mlflow-e2e.sh
+
+fuseml-install:
+	@./dist/fuseml-installer version
```

Comment:
Why the version call?

Reply:
just for tracing purposes, I have moved it out of the makefile to the job.